docs/source/install-tutorial.rst

```diff
@@ -115,3 +145,36 @@ Supervisord config i.e. in ``/etc/supervisor/conf.d/bepasty.conf``:
   redirect_stderr = true                                        ; Save stderr in the same log
 
 Finally reload supervisor: `service supervisor reload`
+
+In your browser go to your server address: ``http://bepasty.example.org``
+
+You should now have a running bepasty server on port 80 (HTTP) of your server.
+
+Places to look when troubleshooting:
+
+::
+
+  journalctl -u nginx.service
+  journalctl -u supervisor.service
+  tail /home/bepasty/logs/nginx-access.log
+  tail /home/bepasty/logs/nginx-error.log
+  tail /home/bepasty/logs/gunicorn_supervisor.log
```

Comment:
never use tail to output logs to your console, that is dangerous. better: `less +F filename`

Reply:
OK - point taken.  Thanks for pointing this out.